gremlin-server/src/test/java/org/apache/tinkerpop/gremlin/server/GremlinServerIntegrateTest.java

```diff
@@ -212,11 +208,6 @@ public Settings overrideSettings(final Settings settings) {
                 // Trust ONLY the server cert
                 settings.ssl.trustCertChainFile = SERVER_CRT;
             	break;
-            case "shouldStartWithDefaultSettings":
```

Comment:
you removed this test configuration option, but left the test itself? did you mean to do that?